scripts/upgrade-react.js

```diff
@@ -0,0 +1,34 @@
+// This file is used to run our nightly cron jobs to test gatsby
```

Comment:
This is a cool script, I thought it would be much more difficult to do this. You make it look easy 😏 Awesome job! :clap: